src/static/clippy/1-tutor.md

```diff
@@ -20,6 +20,9 @@ The idea behind Tridactyl is to allow you to navigate the web more efficiently w
     -   This is most often used for following links, but it has many other submodes.
     -   You can enter this mode with `f` and exit it with `Escape` or `Enter`.
     -   Hint characters are displayed as uppercase letters, but you should type the lowercase letter.
+-   Visual mode
```

Comment:
I think it's worth mentioning that the mode is experimental. I don't know how much you use it but I find that it is quite janky. I often get booted out of the mode when changing the selection, even when there is still a selection.

Reply:
I have been using it more and more recently and yes - I do get kicked out occasionally.